src/filter.js

```diff
@@ -2,6 +2,7 @@ define(function (require) {
   'use strict';
 
   var p5sound = require('master');
+  var Effect = require('effect');
```

Comment:
You can change all "`p5.Effect`" in this module to "`Effect`" (once you add "return p5.Effect" to the Effect module)